### PR TITLE
Component based refactoring

### PR DESCRIPTION
**Pull request by obasajujoshua31**
- [ x] refractored some code

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
 
-class App extends Component {
-    render() {
+function App() {
+    const renderComponents = routes.map(route =>
+        <Route
+            path={route.path}
+            component={route.component}
+            exact={route.exact}
+            key={route.path}
```
Comment (Daymorelah) [inline src/components/App.js]: Would setting the key attribute to a boolean not throw an error to the console?

File: src/components/Home/InsideBusiness/InsideBusinessFooter.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                        <a href="/">42</a>
+                    </li>
+                    <li>
+                        <a href="/">Next Page</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    )
+}
```
Comment (Daymorelah) [inline src/components/Home/InsideBusiness/InsideBusinessFooter.js]: Add new line here.

File: src/components/Home/InsideBusiness/InsideBusinessHeader.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from "react"
+
+
+export default function InsideBusinessHeader() {
+    return ( <div className="section-title">
+        <h3>Inside Business</h3>
+    </div>)
+}
```
Comment (Daymorelah) [inline src/components/Home/InsideBusiness/InsideBusinessHeader.js]: Add new line.

File: src/components/Template.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        <div>
+            <PriceBar />
+            <Header />
+            {props.children}
+            <Footer />
+        </div>
+    )
+}
+
+export default ApplicationTemplate
```
Comment (Daymorelah) [inline src/components/Template.js]: Add new line.

File: src/components/routes.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import Home from "./Home/Home";
+
+
+const routes = [{
+    path: "/",
+    component: Home,
+    exact: true
+}];
+
+export default routes;
```
Comment (Daymorelah) [inline src/components/routes.js]: Add new line.

File: src/components/Home/FeaturedSideNews.js [hunk 6]
```diff
@@ -7,15 +7,15 @@ export default props => {
...
+            {visual_news.map((vn, i) => {
                 return (
                     vn.image_type === 'size450x330' && (
                         <img
                             src={vn.image_file}
                             width="450"
                             height="330"
                             alt=""
-                            key={vn.id}
+                            key={i}
```
[the remark this answers is not in the document]
Comment (obasajujoshua31) [reply src/components/Home/FeaturedSideNews.js]: the other implementation is making the test to fail. the Index i is unique

File: src/components/App.js (hunk 1 above)
Comment (obasajujoshua31) [reply src/components/App.js]: path is not boolean

**Comment by Daymorelah** [comment]
LGTM!

**Pull request merged**